ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -133,6 +133,14 @@
   @Default("1")
   public final int storeMinLogSegmentCountToReclaimToTriggerCompaction;
 
+  /**
+   * The name of the folder where the reserve disk segment pool will be.
+   * A directory with this name will be created at the root of each disk.
+   */
+  @Config("store.reserve.file.dir.name")
```

Comment:
Added a DiskManagerConfig. Anything that is set on the disk level can go here in the future.